addons/im_livechat/models/mail_channel.py

```diff
@@ -85,6 +85,7 @@ def channel_info(self, extra_info=False):
                 if channel.livechat_operator_id:
                     channel_infos_dict[channel.id]['operator_pid'] = channel.livechat_operator_id.with_context(im_livechat_use_username=True).name_get()[0]
                 # add the anonymous or partner name
+                channel_infos_dict[channel.id]['livechat_website_visitor'] = channel._channel_get_livechat_website_visitor_info()
```

Comment:
avoid "website" word in non-website dependencies

Reply:
I hesitate to use `direct_partners` (or `direct_partner`, and then also use this key in `mail/channel_info`), as we basically use this info as direct partner of livechat. What do you think?

Reply:
As discussed yesterday, whatever we do towards supporting multiple anonymous is good.

I wouldn't use "direct" partners as name, because that word doesn't really make sense, especially when there is no "indirect" partners as opposition.

But honestly I think this is fine like it is now, not an array because we don't support it yet but we can easily change it later, and just rename to `livechat_visitor` without the website name.